blocks/footer/footer.css

```diff
@@ -89,7 +92,7 @@ footer ul li {
 }
 
 .ft-social .ft-social-list:nth-child(2) {
-  padding: 10px 0 0 40px;
+  padding: 10px 0 0;
```

Comment:
I think this should be 15px. Please check the alignment of second item with rest of the items. It's not aligned properly.